src/components/menu/MenuState.tsx

```diff
@@ -0,0 +1,58 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+
+export interface IMenuStateConfig {
+    opened: boolean;
```

Comment:
adding `| null` would make it more obvious that `null` is one of expected values

Reply:
Actually the opened prop is optional so i changed it to more correct `opened?: boolean`